modules/apps/29-fee/client/cli/tx.go

```diff
@@ -97,3 +97,28 @@ func NewPayPacketFeeAsyncTxCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewRegisterCounterpartyAddress returns the command to create a MsgRegisterCounterpartyAddress
+func NewRegisterCounterpartyAddress() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "register-counterparty [address] [counterparty-address] [channel-id]",
+		Short:   "Register a counterparty relayer address on a given channel.",
+		Long:    strings.TrimSpace(`Register a counterparty relayer address on a given channel.`),
+		Example: fmt.Sprintf("%s tx ibc-fee register-counterparty cosmos1rsp837a4kvtgp2m4uqzdge0zzu6efqgucm0qdh cosmoss1sp921a4tttgpln6rqhdqe0zzu6efqgucm0qdh channel-0", version.AppName),
```

Comment:
nit: can we have a different bech32 prefix on the counterparty. Use an osmo address as an example

Reply:
good point